wsd/FileServer.cpp

```diff
@@ -984,9 +984,9 @@ void FileServerRequestHandler::preprocessFile(const HTTPRequest& request,
     }
 #endif
 
-    Poco::replaceInPlace(preprocess, std::string("<!--%BRANDING_CSS%-->"), brandCSS);
-    Poco::replaceInPlace(preprocess, std::string("<!--%BRANDING_JS%-->"), brandJS);
-    Poco::replaceInPlace(preprocess, std::string("<!--%CSS_VARIABLES%-->"), cssVarsToStyle(cssVars));
+    replaceVar["BRANDING_CSS"] = brandCSS;
+    replaceVar["BRANDING_JS"] = brandJS;
+    replaceVar["CSS_VARIABLES"] = cssVarsToStyle(cssVars);
```

Comment:
Shouldn't we retain the "<!--%" and "%-->" as well? We then wouldn't need to special case in the replace loop below.